### DISCUSSION TITLE
Any plans to add this container to Unraid's Community Applications?

### DISCUSSION THREAD

**Discussion by jeremytodd1** [lloesche/valheim-server-docker#80] [Ideas]
As someone who doesn't understand the basics of Docker containers but has an Unraid server, I'd love to use this docker container but am not sure how to set it up.

I'd love for this container to be added to Unraid's Community Apps so I could easily set this up with basically a push of a button (after changing the server name, map name, etc. variables, of course.)

Is there any plan to add this to the Community Apps page in Unraid? It'd be very much appreciated.

**Comment by lloesche** [comment]
That's a great idea. I'm not an unraid user myself but would support anyone who would like to port it to the unraid platform. I'm assuming Unraid supports Docker containers with some unraid specific application metadata added to it.

**Comment by armanatz** [comment]
@jeremytodd1 Have you looked into https://forums.unraid.net/topic/79530-support-ich777-gameserver-dockers/ ?

**Reply by jeremytodd1** [reply]
I have, actually. I ended up not having luck with that one. I was able to connect to it locally but never with an external source. I documented my troubleshooting steps and included screenshots over a couple pages starting with this post:

https://forums.unraid.net/topic/79530-support-ich777-gameserver-dockers/?do=findComment&comment=947285

I never ended up getting that container to work even after trying quite a few things.
